raft/rawnode.go

```diff
@@ -123,6 +123,15 @@ func (rn *RawNode) Tick() {
 	rn.raft.tick()
 }
 
+// TickQuiesced advances the internal logical clock by a single tick without
+// performing any other state machine processing. It allows the caller to avoid
+// periodic heartbeats and elections when all of the peers in a Raft group are
+// known to be at the same state. Expected usage is to periodically invoke Tick
+// or TickQuiesced depending on whether the group is "active" or "quiesced".
```

Comment:
I understand this is useful for system like cockroach. But still this seems to be dangerous. Probably add a note to tell users to be sure that they really need this before consider using it.